vagrant/postchecks_qat_driver.sh

```diff
@@ -16,22 +16,30 @@ echo "QAT driver validation"
 
 # Driver validation
 supported_dev="c6xx dh895xcc"
-qat_svc=$(sudo service qat_service status | grep "There is .* QAT acceleration device(s) in the system:")
+qat_svc=$(sudo /etc/init.d/qat_service status | grep "There is .* QAT acceleration device(s) in the system:")
 if [[ "$qat_svc" != *"0"* ]]; then
-    sudo pip install crudini --quiet
-    devices=$(sudo service qat_service status | grep 'state: up')
-    while read -r device; do
-        dev_type=$(echo "$device" | awk '{print $4}' | tr -d ',')
-        if [[ "$supported_dev" != *"$dev_type"* ]]; then
-            echo "The $dev_type device type is not supported by QAT envoy plugin"
-            exit 1
-        fi
-        dev_number=$(echo "$device" | awk '{print $1}' | tr -d 'qat')
-        for ssl_key in NumProcesses NumberCyInstances NumberDcInstances; do
-            sudo crudini --get --existing "/etc/$dev_type$dev_number.conf" SSL $ssl_key > /dev/null
-        done
-    done <<< "$devices"
+    # shellcheck disable=SC1091
+    source /etc/os-release || source /usr/lib/os-release
+    case ${ID,,} in
+        rhel|centos|fedora)
+            sudo pip install crudini --quiet
+            devices=$(sudo /etc/init.d/qat_service status | grep 'state: up')
+            while read -r device; do
+                dev_type=$(echo "$device" | awk '{print $4}' | tr -d ',')
+                if [[ "$supported_dev" != *"$dev_type"* ]]; then
+                    echo "The $dev_type device type is not supported by QAT envoy plugin"
+                    exit 1
+                fi
+                dev_number=$(echo "$device" | awk '{print $1}' | tr -d 'qat')
+                for ssl_key in NumProcesses NumberCyInstances NumberDcInstances; do
+                    sudo crudini --get --existing "/etc/$dev_type$dev_number.conf" SSL $ssl_key > /dev/null
```

Comment:
this is makes sense for the "default" install but our setup cannot use the default configs.

Reply:
The device plugin when run with `-mode kernel` does the checking. If it's running OK and the node has QAT resources registered, I think we have sufficient testing in place (and it works on all distros).

I'd propose we drop `crudini` altogether

Reply:
Fixed on 0b95b8d9aef3d69cfa01828c48025547929fff40 and 8d77c25655aa6c21cbdfe98ba3dd242e90cef454